unittests/lit.cfg.py

```diff
@@ -0,0 +1,27 @@
+# -*- Python -*-
```

Comment:
This probably needs the copyright boilerplate (just run `./utils/boilerplate/all.py`).

Reply:
Done.

{code}
$ figlet

Command 'figlet' not found, but can be installed with:

sudo apt install figlet
sudo apt install toilet
{code}

LOL!

Reply:
:-)